apps/api/src/domain/models/context/note-queries.fetch-by-id.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { INestApplication } from '@nestjs/common';
+import { ArangoDatabaseProvider } from '../../../persistence/database/database.provider';
+import generateDatabaseNameForTestSuite from '../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import TestRepositoryProvider from '../../../persistence/repositories/__tests__/TestRepositoryProvider';
+import { AggregateId } from '../../types/AggregateId';
+import buildDummyUuid from '../__tests__/utilities/buildDummyUuid';
+import { INoteQueryRepository } from './repositories/note-query-repository.interface';
+
+const indexEndpoint = 'resources/notes';
```

Comment:
This should be `web-of-knowledge/all`. See the requirements in the [Jira story](https://coscrad.atlassian.net/browse/CWEBJIRA-166) here.